docs/utils/SolidityLexer.py

```diff
@@ -68,16 +67,11 @@ class SolidityLexer(RegexLexer):
                  ['fixed%dx%d' % ((i), (j + 8)) for i in range(0, 256, 8) for j in range(0, 256 - i, 8)]
              ) + r')\b', Keyword.Type, 'slashstartsregex'),
             (r'(wei|szabo|finney|ether|seconds|minutes|hours|days|weeks|years)\b', Keyword.Type, 'slashstartsregex'),
-            (r'(abstract|boolean|byte|char|class|const|debugger|double|enum|export|'
-             r'extends|final|float|goto|implements|int|interface|long|native|'
-             r'package|private|protected|public|short|static|super|synchronized|throws|'
-             r'transient|volatile)\b', Keyword.Reserved),
-            (r'(true|false|null|NaN|Infinity|undefined)\b', Keyword.Constant),
-            (r'(Array|Boolean|Date|Error|Function|Math|netscape|'
-             r'Number|Object|Packages|RegExp|String|sun|decodeURI|'
-             r'decodeURIComponent|encodeURI|encodeURIComponent|'
-             r'Error|eval|isFinite|isNaN|parseFloat|parseInt|document|this|'
-             r'window)\b', Name.Builtin),
+            (r'(abstract|after|case|catch|default|final|in|inline|interface|let|match|'
+             r'null|of|pure|relocatable|static|switch|try|type|typeof|view)\b', Keyword.Reserved),
+            (r'(true|false)\b', Keyword.Constant),
+            (r'(assert|addmod|block|ecrecover|keccak256|log[0-4]|msg|mulmod|now|'
```

Comment:
Can you please order this the same way as in `GlobalContext.cpp`? (We should change `GlobalContext.cpp` to be in lexical order at some point, but makes it easier to confirm.)